.github/COMMITTING.md

```diff
@@ -0,0 +1,39 @@
+# Committing to Nemo
+
+## Becoming a Committer
```

Comment:
This is for contributor

Reply:
We actually have [this](https://github.com/apache/incubator-nemo/blob/master/.github/CONTRIBUTING.md) already. I've actually referred to the [Spark committer guide](https://spark.apache.org/committers.html). If this seems unrelevant, I could take it off.